### PR TITLE
Aggregate score stats APIs

### PR DESCRIPTION
**Pull request by cmyui**

### CODE REVIEW

**Comment by 7ez** [comment]
this time its the right repo!!!!

**Comment by cmyui** [comment]
For https://github.com/osuAkatsuki/akatsuki-web/pull/46

File: app/usecases/aggregate_score_stats.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+async def total_scores_set() -> int:
+    return sum(
+        [
+            await app.state.services.database.fetch_val(
+                f"""\
+                SELECT count(*)
+                FROM {table}
+                """,
+            )
```
Comment (cmyui) [inline app/usecases/aggregate_score_stats.py:17]:
```suggestion
            ) or 0
```

File: .github/workflows/production-deploy.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - aggregate-apis
```
Comment (tsunyoku) [inline .github/workflows/production-deploy.yml]:
```suggestion
```

**Review by tsunyoku — DISMISSED** [review]

**Review by tsunyoku — APPROVED** [review]

**Pull request merged**